Utilities/JIT.cpp

```diff
@@ -338,6 +338,7 @@ struct MemoryManager1 : llvm::RTDyldMemoryManager
 
 		if (!addr)
 		{
+			jit_log.fatal("JIT: Function '%s' linked but not found.", name);
```

Comment:
JIT: is already logged as channel name
```suggestion
			jit_log.fatal("Function '%s' linked but not found.", name);
```